deploy/charts/trust-manager/Chart.yaml

```diff
@@ -6,7 +6,7 @@ description: trust-manager is the easiest way to manage TLS trust bundles in Kub
 
 home: https://github.com/cert-manager/trust-manager
 
-kubeVersion: ">= 1.25.0"
+kubeVersion: ">= 1.25.0-0"
```

Comment:
Hey, would it be possible to add a comment here to explain why `-0` is required? Otherwise we might remove it again in the future 😅

Reply:
duplicate: https://github.com/cert-manager/trust-manager/pull/182#discussion_r1334164294

Reply:
Done